src/smc.jl

```diff
@@ -114,7 +114,7 @@ function AbstractMCMC.step(
     particles = ParticleContainer(x)
 
     # Perform a particle sweep.
-    logevidence = sweep!(rng, particles, sampler.resampler)
+    logevidence = sweep!(rng, particles, sampler.resampler, particles.vals[nparticles])
```

Comment:
Should also fix https://github.com/TuringLang/Turing.jl/blob/57e0512e3b63e4622c755dc6d933b51eb76d4483/src/inference/AdvancedSMC.jl#L304 when a new release of AdvancedPS is available.